src/zen/kbs/ZenKeyboardShortcuts.mjs

```diff
@@ -993,6 +994,21 @@ class nsZenKeyboardShortcutsVersioner {
         }
       }
     }
+    if (version < 10) {
+      // Migrate from version 9 to 10
+      // Add the new pin/unpin tab toggle shortcut (no default key assigned)
+      data.push(
+        new KeyShortcut(
+          'zen-toggle-pin-tab',
```

Comment:
Set this shortcut to `ctrl+shift+D` and remove the current shortcut for `ctrl+shift+D` that firefox has implemented.